packages/account-v2/src/modules/IDVForm/utils.ts

```diff
@@ -0,0 +1,98 @@
+import * as Yup from 'yup';
+import { AnyObject } from 'yup/lib/object';
+
+export const getExampleFormat = (example_format?: string) => (example_format ? `Example: ${example_format}` : '');
+
+export type TDocument = {
+    additional?: {
+        display_name?: string;
+        example_format?: string;
+        format?: string;
+    };
+    example_format?: string;
+    id: string;
+    text: string;
+    value: string;
+};
+
+const validateDocumentNumber = (
+    documentConfig: TDocument | undefined,
+    documentNumber: string,
+    context: Yup.TestContext<AnyObject>
+) => {
+    const isNotSameAsExample = documentNumber === documentConfig?.example_format;
```

Comment:
Here `isNotSameAsExample` check if `documentNumber === documentConfig?.example_format;`🤔
is this correct?

Reply:
Well the condition is correct. If the name is confusing, I'll change it